tests/test_backend.py

```diff
@@ -659,6 +659,43 @@ def test_get(self):
         self.assertEqual(self.cache.get("key", default="default"), "default")
 
 
+class DjangoOmitExceptionsPriority1Tests(unittest.TestCase):
+    def setUp(self):
+        self._orig_setting = django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS
+        django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS = False
+        caches_setting = copy.deepcopy(settings.CACHES)
+        caches_setting["doesnotexist"]["IGNORE_EXCEPTIONS"] = True
+        cm = override_settings(CACHES=caches_setting)
+        cm.enable()
+        self.addCleanup(cm.disable)
+        self.cache = caches["doesnotexist"]
+
+    def tearDown(self):
+        django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS = self._orig_setting
+
+    def test_get(self):
+        self.assertIsNone(self.cache.get("key"))
+
+
+class DjangoOmitExceptionsPriority2Tests(unittest.TestCase):
+    def setUp(self):
+        self._orig_setting = django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS
+        django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS = True
+        caches_setting = copy.deepcopy(settings.CACHES)
+        caches_setting["doesnotexist"]["IGNORE_EXCEPTIONS"] = False
+        cm = override_settings(CACHES=caches_setting)
+        cm.enable()
+        self.addCleanup(cm.disable)
+        self.cache = caches["doesnotexist"]
+
+    def tearDown(self):
+        django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS = self._orig_setting
+
+    def test_get(self):
+        with self.assertRaises(KeyError):
```

Comment:
This test fails for me on master as a `KeyError` is never raised.

Further, the cache should *never* raise a `KeyError`. From the [Django docs](https://docs.djangoproject.com/en/2.1/topics/cache/#django.core.caches.cache.get). So, for this test to be valid, I think this should be expecting some other kind of exception.